src/Console/Command/DescribeCommand.php

```diff
@@ -225,7 +225,13 @@ private function describeRule(OutputInterface $output, $name)
                 $old = $codeSample->getCode();
                 $tokens = Tokens::fromCode($old);
                 if ($fixer instanceof ConfigurableFixerInterface) {
-                    $fixer->configure($codeSample->getConfiguration());
+                    $configuration = $codeSample->getConfiguration();
+
+                    if (null === $configuration) {
```

Comment:
isn't this an error in the configuration of the sample itself? i.e. shouldn't we just throw an exception here?